studiopress/atomic-blocks
# Add pricing table two section design

**Summary of change:**
This PR adds another pricing section design to the layout modal. We'll probably want a better naming convention than 'Pricing Table Two'

**This PR has been:**

**How to test:**
Add the layout block, select the Pricing Table Two section and visually test it.

**See:** #186 

**Suggested Changelog Entry:**
Add Pricing Table Two section design.
